.github/workflows/ssl-postgres-unit-tests.yaml

```diff
@@ -0,0 +1,16 @@
+name: Unit Tests with External SSL Postgres
+on: [push, pull_request]
+
+jobs:
+  run-unit-tests-ssl-postgres:
+    runs-on: ubuntu-latest
+    timeout-minutes: 90
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
+      cancel-in-progress: true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Run Unit Tests with External SSL Postgres
+        run: make test-external-postgres
```

Comment:
why do we need a different workflow? can't we use the sanity workflow and run both make test-postgres & make test-external-postgres

Reply:
why not? running one after the other will make it really lengthy... is it something we want to avoid? (limited resources in github?). let's have a discussion about it.

Reply:
Decided to leave for now only test-external-pg-sanity. The unit test will run only on default Postgres for now.